sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -1724,6 +1722,13 @@ private void updatePersonAndCaseByOutcome(CaseDataDto existingCase, Case newCase
 					PersonDto existingPerson = PersonFacadeEjb.toDto(newCase.getPerson());
 					newCase.getPerson().setPresentCondition(PresentCondition.ALIVE);
 					// attention: this may lead to infinite recursion when not properly implemented
+					personFacade.onPersonChanged(existingPerson, newCase.getPerson());
+				}
+			}else if(newCase.getOutcome()==CaseOutcome.UNKNOWN) {
```

Comment:
This code doesn't make much sense to me. Just because the outcome of a case is unknown doesn't mean that the present condition also necessarily has to be unknown. It could just mean that it's not known whether the person has already recovered or not, but is definitely alive. Please remove this change.